src/Core/Pages/ShellPage.razor

```diff
@@ -7,6 +7,7 @@
         </StackLayout>
     </FlyoutHeader>
     <ChildContent>
+        <StyleSheet Resource="App.css" Assembly="GetType().Assembly" />
```

Comment:
Would this also work for `Label.app-menu-header` in the `FlyoutHeader` (which was my problem before)?

Reply:
In Xamarin-based apps (which Mobile Blazor Bindings apps are), CSS is global to the app. So if your CSS has a style it will apply everywhere that it is possible to apply. The exact location of the `<StyleSheet ... />` declaration doesn't matter as long as:

1. It's a valid location to begin with. It must be contained within an element that derives from `View`. You can find the official list here and zoom in on the chart: https://docs.microsoft.com/xamarin/xamarin-forms/internals/class-hierarchy. You'll see that most controls derive from `View` so it's usually not hard to find a place to put it.
2. It needs to be in a section that will actually load. For example, if the `<StyleSheet ... />` declaration is inside a code block such as `@if (false) { <StyleSheet ... /> }` then it won't render, and thus no CSS will load.

For these reasons, putting it inside the `ChildContent` of the `Shell` control is perfectly safe. `Shell` ultimately derives from `View`, and the `ChildContent` of the `Shell` is always loaded.

Reply:
Oh, and the answer to your question: Yes, it should work! 😄

Reply:
Thanks for that detailed answer!